glean-core/rlb/src/dispatcher/global.rs

```diff
@@ -7,7 +7,7 @@ use std::sync::RwLock;
 
 use super::{DispatchError, DispatchGuard, Dispatcher};
 
-pub const GLOBAL_DISPATCHER_LIMIT: usize = 100;
+pub const GLOBAL_DISPATCHER_LIMIT: usize = 1000;
```

Comment:
There's a test for this: `glean-core/rlb/tests/overflowing_preinit.rs`

Reply:
Well, poo. Shows me for just redressing a quick'n'dirty patch as a proper change.